ckan/plugins/toolkit.py

```diff
@@ -290,6 +280,22 @@ def _initialize(self):
         t['HelperError'] = HelperError
         t['enqueue_job'] = enqueue_job
 
+        if six.PY2:
+
+            t['literal'] = webhelpers.html.tags.literal
+            t['response'] = pylons.response
+            self.docstring_overrides['response'] = '''The Pylons response object.
+
+Pylons uses this object to generate the HTTP response it returns to the web
+browser. It has attributes like the HTTP status code, the response headers,
+content type, cookies, etc.
+
+'''
+            t['BaseController'] = base.BaseController
+            # TODO: Sort these out
```

Comment:
What's to sort out about these?

Reply:
These are the CLI methods. I wasn't clear how extensions will extend the new Click based CLI interface, so "Sort these out" actually means "Check if we need to load some of these in py3". I'll add a note on https://github.com/ckan/ckan/pull/5112 to clarify this

Reply:
👍